index.html

```diff
@@ -66,24 +66,31 @@ <h2>Data Management</h2>
       <div class="category">
         <hr>
         <h2>IoT (Internet of Things)</h2>
-  
+
         <div class="listing">
           <h3><a href="https://github.com/spyderboy/WYEP-Alexa-Skill">Alexa Skill for Music Stations</a>: An example Alexa Skill for playing live streams and calling Composer API to identify song info</h3>
           <p>This is a simple skill that demonstrates live music streaming and interacting with the NPR Composer API, initially developed for WYEP</p>
         </div>
 
         <div class="listing">
-          <h3><a href="https://github.com/APMG/mpr-alexa-base">Alexa Skill for Music and News stations</a>: And example Alexa skill for radio stations with features requiring user authentication.</h3>
+          <h3><a href="https://github.com/APMG/skill-share">Alexa Skill for Music and News stations</a>: And example Alexa skill for radio stations with features requiring user authentication.</h3>
```

Comment:
@eileennoonan83 is this part intended? It changes the link from https://github.com/APMG/mpr-alexa-base to https://github.com/APMG/skill-share for this entry.  Same for line 78?

Reply:
It is intended. This is basically intended to be a better / more generalized version of `mpr-alexa-base`

Reply:
I have updated this with a new commit that I think is clearer about what's going on.